docs/components/callout/index.md

```diff
@@ -2,13 +2,14 @@
   import iOS from './ios.md';
   import data from './data.json';
   import { mapFrameworkStatuses } from '../utils.js';
+  import android from './android.md';
 </script>
 
 # Callout
 
 {{ data.description }}
 
-::: warning iOS only
+::: Warning! Native only!
```

Comment:
```suggestion
::: Warning Native only!
```
You need to remove the exclamation mark, otherwise it will not be styled as a warning box. :)